django/db/models/sql/compiler.py

```diff
@@ -283,6 +285,9 @@ def get_select(self, with_col_aliases=False):
                 # Reference to a column.
                 elif isinstance(expression, int):
                     expression = cols[expression]
+                # ColPairs cannot be aliased.
+                if isinstance(expression, ColPairs):
+                    alias = None
```

Comment:
Yes, this will compile `ColPairs` as:

```sql
SELECT "composite_pk_user"."tenant_id", "composite_pk_user"."id" FROM ...
```

instead of:

```sql
SELECT "composite_pk_user"."tenant_id", "composite_pk_user"."id" AS "pk" FROM ...
```

Reply:
makes sense thanks!